src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTarget.cs

```diff
@@ -61,5 +61,15 @@ public override int GetHashCode()
 
             return combiner.CombinedHash;
         }
+
+        public LockFileTarget Clone()
```

Comment:
Same as before. 
This is not a true clone and you need tests.

Reply:
wil get rid of it